WalletWasabi.Fluent/ViewModels/Wallets/Home/History/Details/CoinJoinDetailsViewModel.cs

```diff
@@ -0,0 +1,64 @@
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using System.Windows.Input;
+using Avalonia;
+using NBitcoin;
+using ReactiveUI;
+using WalletWasabi.Fluent.Extensions;
+using WalletWasabi.Fluent.ViewModels.Navigation;
+using WalletWasabi.Fluent.ViewModels.Wallets.Home.History.HistoryItems;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets.Home.History.Details;
+
+[NavigationMetaData(Title = "Coinjoin Details")]
+public partial class CoinJoinDetailsViewModel : RoutableViewModel
+{
+	private readonly CoinJoinHistoryItemViewModel _coinJoin;
+
+	[AutoNotify] private string _date = "";
+	[AutoNotify] private string _status = "";
+	[AutoNotify] private Money? _coinJoinFee;
+	[AutoNotify] private string _coinJoinFeeString = "";
+	[AutoNotify] private uint256? _transactionId;
+
+	public CoinJoinDetailsViewModel(CoinJoinHistoryItemViewModel coinJoin)
+	{
+		_coinJoin = coinJoin;
+
+		SetupCancel(enableCancel: false, enableCancelOnEscape: true, enableCancelOnPressed: true);
+		NextCommand = CancelCommand;
+
+		CopyCommand = ReactiveCommand.CreateFromTask<uint256>(async txid =>
+		{
+			if (Application.Current is { Clipboard: { } clipboard })
+			{
+				await clipboard.SetTextAsync(txid.ToString());
+			}
+		});
+
+		Update();
+	}
+
+	public ICommand CopyCommand { get; }
+
+	protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disposables)
+	{
+		base.OnNavigatedTo(isInHistory, disposables);
+
+		Observable
+			.FromEventPattern(_coinJoin, nameof(PropertyChanged))
```

Comment:
This won't work. The items in the history list don't get updated anymore their properties will never change. The whole list got replaced on trigger.

`_balanceChange` Observable should be the trigger here.

Reply:
Added observable trigger